src/ray/test/run_core_worker_tests.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 set -x
 
-bazel build "//:core_worker_test" "//:mock_worker"  "//:raylet" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
+bazel build -c dbg --config=asan "//:core_worker_test" "//:mock_worker" "//:raylet" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
```

Comment:
running debug build might be very slow, is this required by asan?

Reply:
Do debug builds work at all (given the `DEBUG` issue)?